deb/publish.go

```diff
@@ -629,6 +660,38 @@ func (p *PublishedRepo) Publish(packagePool aptly.PackagePool, publishedStorageP
 			}
 		}
 
+		// AppStream file colection
+		appstream, err := p.GetAppStream(component)
+		if err != nil {
+			return fmt.Errorf("Unable to get AppStream files: %s", err)
+		}
+
+		if len(appstream) > 0 {
+			if progress != nil {
+				progress.Printf("Copying appstream files...\n")
```

Comment:
too bad progress isn't a struct, we could have gotten right of all this null checks